tests/test_reactions.py

```diff
@@ -1,11 +1,13 @@
+import pytest
 from autode.reactions.reaction_types import classify, Addition
 from autode.exceptions import ReactionFormationFailed
-import pytest
 
 
 def test_classify():
-    # Can do this with integers are in classify there is no check on their
-    # contents, only their length
+    """
+    Testing with integers as in classify there is no check on the type
```

Comment:
Feel like this needs a comma after integers? I had to read a couple of times to understand